test/integration-test/src/tests/rbpf.rs

```diff
@@ -48,7 +48,7 @@ fn use_map_with_rbpf() {
     // - Note that rbpf does not convert fds into real pointers,
```

Comment:
the comment above here needs an update? maybe you can generalize it so it's not so tightly coupled to the code

Reply:
Updated and gave generalizing a shot